mscore/pluginCreator.cpp

```diff
@@ -430,19 +436,20 @@ void PluginCreator::load()
 //   savePlugin
 //---------------------------------------------------------
 
-void PluginCreator::savePlugin()
+void PluginCreator::doSavePlugin(bool saveas) 
       {
-      if (created) {
+      if (saveas) {
             path = mscore->getPluginFilename(false);
             if (path.isEmpty())
                   return;
             }
       QFile f(path);
       QFileInfo fi(f);
+      msg("Path="+path+"\n");
```

Comment:
Just curious: what is this about?

Reply:
Originally my own debugging, but it does no harm and tells you what is going on, ie, where the file is save. 
In Musescore 2.1, I found i got into a situation where it refused to save the file I was working on because it had failed to put a .qml extension on it.

Reply:
In any case, I've put the hook into .git\hooks. It does not appear to make any difference to the previous commits.
I also followed your instructions, re: git -- revision.h, and github helpfully close my pull request.
If I do git pull --rebase musescore master, what will it do to my existing code?
Get very twitchy about git because everything I try just seems to be making it worse.

Reply:
the `git checkpout --revision.h` needs to be done before you have it commited. The `git pull --rebase musescore master` should (!) do nothing to your code.